tuxemon/menu/menu.py

```diff
@@ -191,7 +191,12 @@ class Menu(Generic[T], state.State):
     # File to load for image background
     background_filename: Optional[str] = None
     menu_select_sound_filename = "sound_menu_select"
-    font_filename = "PressStart2P.ttf"
+    if prepare.CONFIG.locale == "zh_CN":
```

Comment:
Can you make a function to get the filename?

Reply:
Or load it into a global const variable.

Reply:
The issue was that you have the same code twice, here and in theme.

This change is a mistake because you call prepare.fetch again within the startup method.


I was asking for:
font_filename = get_font_filename(prepare.CONFIG.locale)

or 

font_filename = A_CONST_WE_IMPORTED_FROM_SOMEWHERE

Reply:
ok